docs/i18n-disabled/en/docusaurus-plugin-content-docs/current/developers/sdk/Tracks.md

```diff
@@ -293,7 +293,7 @@ Create an object with the following fields and pass it as the first argument, as
 | :-------- | :------- | :------------------ | :----------- |
 | `trackId` | `string` | The ID of the track | **Required** |
 
-TODO: should we document premium content signature?
+TODO: should we document stream signature?
```

Comment:
this will now become nft access signature fyi. the access fields are now the more relevant ones, and im thinking they are self explanatory. we can document the nft access signature if we want but i dont see a strong need to atm